include/roboteam_ai/stp/new_constants/ControlConstants.h

```diff
@@ -6,37 +6,44 @@
 #define RTT_CONTROLCONSTANTS_H
 
 namespace rtt::ai::stp::control_constants {
```

Comment:
```suggestion
namespace rtt::control::constants {
```
Theyre not just for STP, right?

Reply:
I thinks this is better, but Jesse made this, so what do you think @RobotJesse ?

Reply:
For now we will leave it like this, when STP is done we will refactor the namespace.